src/Resources/views/Common/Label/refundPaymentState.html.twig

```diff
@@ -0,0 +1,13 @@
+{%
+    set viewOptions = {
+        completed: { icon: 'adjust', color: 'green' },
+        new: { icon: 'clock', color: 'olive' },
+    }
+%}
+
+{% set value = 'sylius.ui.' ~ data %}
```

Comment:
Is it alright to re-use `sylius.ui` translations for `new` and `completed` states? Should I replace it with english only `sylius_refund.ui` values?